todo_app/templates/index.html

```diff
@@ -12,7 +12,12 @@ <h1 class="display-4">To-Do App</h1>
       <h2>Items</h2>
       <ul class="list-group mb-4">
         <!-- List the todo items here -->
+	{% for item in items %}
+               	<li>{{ item.title }}</li>
+	{% endfor %}	
       </ul>
     </div>
+    <!-- Button to go to add item page -->
+    <button onclick="window.location.href='/addItem'">Add a new item</button>
```

Comment:
Why not use an anchor (`<a>`) element? That's all this is - a link - and we want html to be as semantic as possible.
If the reasoning was for styling, you could add `class="btn btn-primary"` to it, since Bootstrap is already being used.